spec/features/users_utilize_search_form_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+feature "visitor sees search bar on index page" do
+  scenario "visitor visits root directory and sees search bar" do
+    visit root_path
+
+    expect(page).to have_button('Search')
+
+
+  end
+end
+
+feature "visitor interacts with search bar" do
+  scenario "visitor does not use search function" do
+    bar1 = FactoryGirl.create(:bar)
+    bar2 = FactoryGirl.create(:bar)
+    visit root_path
+
+    expect(page).to have_content bar1.name
+    expect(page).to have_content bar2.name
+
+  end
+
+  scenario "visitor searches for bar1, and doesn't see bar2" do
+    visit root_path
+    bar1 = FactoryGirl.create(:bar)
```

Comment:
You can refactor the bar1 and bar2 creations in let statements outside your scenario block.